wsclient/src/test/java/org/hpccsystems/ws/client/WSECLTests.java

```diff
@@ -0,0 +1,181 @@
+/*##############################################################################
```

Comment:
@asselitx this is the wsecl test suite, any wsecl related tests would be hosted here